client/network/src/service/out_events.rs

```diff
@@ -191,11 +184,20 @@ impl OutChannels {
 					sender.creation_backtrace,
 					Backtrace::force_capture(),
 				);
+			} else if sender.warning_fired &&
+				current_pending < sender.queue_size_warning.wrapping_div(2)
+			{
+				sender.warning_fired = false;
+				info!(
+					"Channel `{}` is no longer overflowed. Number of events: {}",
+					sender.name, current_pending
+				);
 			}
```

Comment:
Let's remove this. Otherwise it will get really spammy. This is just about saying "hey, there is somehing" and not on/off/on/off/on/off

Reply:
`debug` sounds reasonable